.github/workflows/cd_release.yaml

```diff
@@ -80,6 +80,48 @@ jobs:
             ./install/release/*.dll
 
 
+  release_windows_x86:
+    name: Windows x86
+    runs-on: windows-2022
```

Comment:
If it's running on the 2022 version instead of the 2025 version, it will still have the unicode bug, right?

Reply:
I think that's related to the version of gcc, not the OS that hosted compilation.
As far as I know, the GitHub runner images don't provide 32 bit mingw, so it doesn't matter what OS it uses (though, we should probably have it use 2025 anyway).
This Job installs 32 bit mingw through Chocolatey, which provides gcc 13.2.

Another option would be for us to just download and extract the files from mingw directly, which would enable us to acquire hte latest version.

Reply:
IIRC GCC 14 is the one with the fix, so this version would not get it. I'm not really a fan of adding a new release asset with known issues.

Reply:
Turns out the GitHub hosted Windows runners do come with 32-bit mingw, but we need to add it to the PATH ourselves.
The Windows 2025 runners do indeed have gcc 14.2 32-bit, so we can just use that.
[Here's a test run.](https://github.com/warp-core/endless-sky/actions/runs/13723870838/job/38385528869)